src/FakeItEasy.netstd/project.json

```diff
@@ -2,21 +2,11 @@
   "dependencies": {
     "Castle.Core": "4.0.0-beta001",
     "Microsoft.Extensions.DependencyModel": "1.0.0",
+    "NETStandard.Library":"1.6.0",

```

Comment:
Won't that cause NuGet to fetch far more dependencies than necessary? We only need a small part of what's in NETStandard.Library